common/src/main/java/com/wynntils/models/items/encoding/impl/block/RequirementsDataTransformer.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.encoding.impl.block;
+
+import com.wynntils.models.character.type.ClassType;
+import com.wynntils.models.elements.type.Skill;
+import com.wynntils.models.gear.type.GearRequirements;
+import com.wynntils.models.items.encoding.data.RequirementsData;
+import com.wynntils.models.items.encoding.type.DataTransformer;
+import com.wynntils.models.items.encoding.type.ItemTransformingVersion;
+import com.wynntils.utils.UnsignedByteUtils;
+import com.wynntils.utils.type.ArrayReader;
+import com.wynntils.utils.type.ErrorOr;
+import com.wynntils.utils.type.Pair;
+import com.wynntils.utils.type.UnsignedByte;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class RequirementsDataTransformer extends DataTransformer<RequirementsData> {
+    private static final List<Pair<ClassType, Integer>> CLASS_TYPE_IDS = List.of(
+            Pair.of(ClassType.NONE, 0),
```

Comment:
Same question here, maybe put the id in ClassType?

Reply:
I'm not sure why this was resolved..?

Reply:
To clarify, I mean that ClassType should have an `encodingId` field containing these numbers.